### ISSUE TITLE
Depth Below Horizon fails with "write_output() got an unexpected keyword argument 'fill'"

### ISSUE THREAD

**Issue by anitagraser** [zoran-cuckovic/QGIS-visibility-analysis#23]
The output raster is not correctly generated:

```
Traceback (most recent call last):
File "C:/Users/agraser/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\ViewshedAnalysis\algorithms\viewshed_horizon_depth.py", line 175, in processAlgorithm
dem.write_output(output_path, fill = np.nan)
TypeError: write_output() got an unexpected keyword argument 'fill'
```

**Comment by zoran-cuckovic** [comment]
Thank you for signalling the issue! I've fixed the problem in the [experimental branch](https://github.com/zoran-cuckovic/QGIS-visibility-analysis/tree/experimental). I'll update QGIS plugin repository shortly.

**Comment by zoran-cuckovic** [comment]
Fixed in 0.6.6 version
